python-checks/src/main/resources/org/sonar/l10n/py/rules/python/Sonar_way_profile.json

```diff
@@ -20,6 +20,7 @@
     "S1045",
     "S1066",
     "S1110",
+    "S1128",
```

Comment:
This rule shouldn't be in SonarWay profile. We forgot to update RSPEC

Reply:
I created a PR for this: https://github.com/SonarSource/rspec/pull/2772

Reply:
A PR for rspec has been created: https://github.com/SonarSource/rspec/pull/2773

Reply:
Also, it has been fixed manually now for sonar-python in this branch. Same for S6661